mmpose/datasets/pipelines/pose3d_transform.py

```diff
@@ -115,6 +117,110 @@ def __call__(self, results):
         return results
 
 
+@PIPELINES.register_module()
+class ImageCoordinateNormalization:
+    """Normalize the 2D joint coordinate with image width and height. Range [0,
+    w] is mapped to [-1, 1], while preserving the aspect ratio.
+
+    Args:
+        item (str|list[str]): The name of the pose to normalize.
+        norm_camera (bool): Whether to normalize camera intrinsics.
+            Default: False.
+        camera_param (dict|None): The camera parameter dict. See the camera
+            class definition for more details. If None is given, the camera
+            parameter will be obtained during processing of each data sample
+            with the key "camera_param".
+    Required keys:
+        item
+    Modified keys:
+        item (, camera_param)
+    """
+
+    def __init__(self, item, norm_camera=False, camera_param=None):
+        self.item = item
+        if isinstance(self.item, str):
+            self.item = [self.item]
+
+        self.norm_camera = norm_camera
+
+        if camera_param is None:
+            self.static_camera = False
+        else:
+            self.static_camera = True
+            self.camera_param = camera_param
+
+    def __call__(self, results):
+        center = np.array(
+            [0.5 * results['image_width'], 0.5 * results['image_height']],
+            dtype=np.float32)
+        scale = np.array(0.5 * results['image_width'], dtype=np.float32)
```

Comment:
Should center/scale be calculated from the image shape or from the human bbox (like the provided ones in h36m data).

Reply:
The original implementation uses center/scale calculated from the image shape. Human bbox is not used here.

Reply:
I think this could be confusing to users if there are "center" and "scale" in the results from the dataset but we calculate them here again with a different method.

I suggest we add an argument to the h36m_dataset, e.g. "use_person_bbox", to determine the center/scale are from the bounding box (as provided in the annotation) or from the image (as calculated here). And in this transform, it simple get the scale/center from the result. What do you think?